edk2toollib/uefi/telem/plugins/__init__.py

```diff
@@ -0,0 +1,13 @@
+# @file __init__.py
+# Imports all classes within this directory
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+from os.path import dirname, basename, isfile
+import glob
+
+modules = glob.glob(dirname(__file__)+"/*.py")
```

Comment:
yeah you're not guaranteed to be co-located. Pip files end up in weird spaces. I think maybe you could just have a static list of modules if they need to check it into pytools

Reply:
It seems like the edk2toollib and edk2toolext folders stay intact (just based on looking at my venv folders). Is this not always the case?